subprojects/docs/src/docs/userguide/authoring-builds/build_lifecycle.adoc

```diff
@@ -194,6 +194,10 @@ Task execution includes most of the work you normally associate with a build: do
 
 ==== React to Task Execution Notifications
 
+WARNING: Task execution events are not compatible with the <<configuration_cache.adoc#config_cache:requirements:build_listeners,configuration cache>>.
+You can use <<build_services#build_services,build services>> registered to receive information about <<build_services#operation_listener, task execution>>,
+to receive notifications in a way compatible with the configuration cache.
```

Comment:
I think we're missing a word in this sentence.

Reply:
> I think we're missing a word in this sentence.

Thanks, I rewrote the sentence to make it clearer.